cala-server/src/graphql/account.rs

```diff
@@ -139,6 +140,26 @@ impl Account {
         )
         .await
     }
+
+    async fn entries(
+        &self,
+        ctx: &Context<'_>,
+        from: Timestamp,
+        until: Option<Timestamp>,
```

Comment:
@vindard @bodymindarts this pr adds ranged query for `accounts` and `account_sets`. Should we also include a more generic paginated query as well ?